cebra/models/decoders.py

```diff
@@ -0,0 +1,38 @@
+import torch.nn as nn
```

Comment:
why not have decoders somewhere like integrations? to me model is the encoders only cc @stes